pact-data-model/src/lib.rs

```diff
@@ -55,8 +56,8 @@ pub struct ProductFootprint<T: JsonSchema> {
     pub extensions: Option<Vec<DataModelExtension<T>>>,
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, PartialEq)]
-#[serde(rename_all = "camelCase")]
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, JsonSchema)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
```

Comment:
```suggestion
#[serde(rename_all = "camelCase")]
```

Reply:
Do we want to remove `deny_unknown_fields` though? Without it, random fields can be added to the JSON and it will still pass validation against our schema. Do we want to allow this?